keydates-worker/keydates_worker.py

```
@@ -1437,6 +1559,15 @@ def series_path(series):
         all_rejected += rejected
         if changes or refuted or held:
             log(f"{base}: +{len(changes)} applied, {len(refuted)} refuted, {len(held)} held")
+        # S2: on a total backend outage, grinding through every con (4 retries ×
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)